src/ckb/ckb-info.plist

```diff
@@ -5,9 +5,11 @@
     <key>CFBundleExecutable</key>
     <string>ckb</string>
     <key>CFBundleIconFile</key>
-    <string>ckb-logo.icns</string>
+    <string>@ICON@</string>
```

Comment:
Is this some special qmake magic? I don't see any preprocessing code for qmake that would properly resolve the icon.

Reply:
You need to go read the Qt documentation, i explains how all these variables work.  Tbh, the root of this issue was failure to read the documentation.